deploy/olm-catalog/performance-addon-operator/performance-addon-operator.package.yaml

```diff
@@ -1,7 +1,7 @@
 channels:
 - currentCSV: performance-addon-operator.v4.4.0
-  name: 4.4.0
-- currentCSV: performance-addon-operator.v4.5.0
-  name: 4.5.0
-defaultChannel: 4.5.0
+  name: "4.4"
```

Comment:
Same here, the channel names must not change I think. Btw, I wonder if we need the old release CSV anyway as skipRange probably ignores them and we have no replaces field anymore.

Reply:
AFAIK the channel name is just a random string indeed, but let's get this chance to doublecheck